src/sciline/pipeline.py

```diff
@@ -200,6 +209,13 @@ def _repr_html_(self) -> str:
         nodes = ((key, data) for key, data in self.underlying_graph.nodes.items())
         return pipeline_html_repr(nodes)
 
+    def final_result_keys(self) -> tuple[Key, ...]:
```

Comment:
Sorry about discussing names a second time, but can we think about the name once more. This seems a bit hard to remember, with the 3-part name. Have you considered `output_keys`? It would also be symmetric with the terminology "inputs" or input keys", which we might use elsewhere.

Reply:
No worries, yeah if you think `output_keys` is better we can use that instead :+1:

> Have you considered `output_keys`?

Yes it's one of the names I suggested the first time you brought up naming :laughing: